packages/service-library/src/servicelib/redis.py

```diff
@@ -72,7 +82,16 @@ async def setup(self) -> None:
             f"{self._client=}",
         )
 
+        self._health_check_task = start_periodic_task(
```

Comment:
THOUGHT: Can we integrate this as a mixin instead of enforcing it everyewhere where this client is used. 
Note that after this change suddenly you are creating another periodic task in all services that might not need it. 

Same story with rabbit.

THOUGHT: another approach would be some sort of lazy check, i.e. only when there is a call to redis the class updates `self._is_redis_healthy`

Reply:
after adding a mixin I realised it was not required so I used plain 'ol good inheritance